cve_bin_tool/cli.py

```diff
@@ -32,6 +32,8 @@
 from .cvedb import CVEDB
 from .log import LOGGER
 
+output_csv_file = 0
+fileOpenMode = "w"
```

Comment:
You have to open the file in write mode every time, so this doesn't need to be a variable, let alone a global one.